tacotron2/layers.py

```diff
@@ -48,7 +48,8 @@ def __init__(self, filter_length=1024, hop_length=256, win_length=1024,
         self.sampling_rate = sampling_rate
         self.stft_fn = STFT(filter_length, hop_length, win_length)
         mel_basis = librosa_mel_fn(
-            sampling_rate, filter_length, n_mel_channels, mel_fmin, mel_fmax)
+            sr=sampling_rate, n_fft=filter_length, n_mels=n_mel_channels, 
+            fmin=mel_fmin, fmax=mel_fmax)
```

Comment:
is there a reason for making this explicit in particular?

Reply:
It was just throwing a future warning